src/js/modules/accounts/components/forms/SignInForm.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Field, reduxForm } from "redux-form";
+import injectSheet from "react-jss";
+
+const styles = {
+  errorMessage: {
+    color: "red",
+  },
+};
+
+const validate = values => {
```

Comment:
again, validate what? Also you're using the same regex, pull it out into a constants file